src/Components/Benefits.js

```diff
@@ -0,0 +1,119 @@
+import React from 'react'
```

Comment:
Overall, the code looks well-written and follows best practices of using styled-components to create reusable and maintainable styles for components